src/main.ts

```diff
@@ -102,34 +106,44 @@ async function runTrivy(): Promise<number> {
     };
 
     const trivyToolRunner = new ToolRunner(trivyPath, [imageName], trivyOptions);
-    const trivyStatus = await trivyToolRunner.exec();
+    const trivyStatus = await trivyToolRunner.exec();    
     return trivyStatus;
 }
 
-async function runDockle(): Promise<void> {
-    const docklePath = await getDockle();
-    const dockleEnv = await setDockleEnvVariables();
+async function runDockle(): Promise<number> {
+    const docklePath = await dockleHelper.getDockle();
+    const dockleEnv = await getDockleEnvVariables();
```

Comment:
NIT: If you're not modifying it, you might as well invoke the function wherever you're using it